rllib/tuned_examples/bc/cartpole-bc.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+
+from ray.rllib.algorithms.bc import BCConfig
+from ray.rllib.utils.metrics import (
+    ENV_RUNNER_RESULTS,
+    EPISODE_RETURN_MEAN,
+    EVALUATION_RESULTS,
+    TRAINING_ITERATION_TIMER,
+)
+from ray.rllib.utils.test_utils import (
+    add_rllib_example_script_args,
+    run_rllib_example_script_experiment,
+)
+
+parser = add_rllib_example_script_args()
+# Use `parser` to add your own custom command line options to this script
+# and (if needed) use their values toset up `config` below.
+args = parser.parse_args()
+
+assert (
+    args.env == "CartPole-v1" or args.env is None
+), "This tuned example works only with `CartPole-v1`."
+
+# Define the data paths.
+data_path = "tests/data/cartpole/cartpole-v1_large"
+base_path = Path(__file__).parents[2]
+print(f"base_path={base_path}")
+data_path = "local://" + base_path.joinpath(data_path).as_posix()
+print(f"data_path={data_path}")
+
+# Define the BC config.
+config = (
+    BCConfig()
+    .environment(env="CartPole-v1")
+    .api_stack(
+        enable_rl_module_and_learner=True,
+        enable_env_runner_and_connector_v2=True,
+    )
+    .learners(
```

Comment:
We can remove this method call (`learners(...)`), b/c `run_rllib_example_script_experiment` automates this logic.